packages/ketcher-core/src/chemistry/entities/Struct.ts

```diff
@@ -482,6 +486,11 @@ export class Struct {
     item.pos = pos
   }
 
+  textSetPosition(id: number, position: Vec2): void {
+    const item = this.texts.get(id)!
```

Comment:
why you don't check item on 'undefined'?

Reply:
fixed